discovery/kubernetes/kubernetes.go

```diff
@@ -91,6 +92,26 @@ type SDConfig struct {
 	Role               Role                         `yaml:"role"`
 	HTTPClientConfig   config_util.HTTPClientConfig `yaml:",inline"`
 	NamespaceDiscovery NamespaceDiscovery           `yaml:"namespaces,omitempty"`
+	Selectors          []ResourceSelectorConfigRaw  `yaml:"selectors,omitempty"`
+}
+
+type RoleSelectorConfig struct {
```

Comment:
AFAICT `RoleSelectorConfig` doesn't need to be exported and it doesn't need "yaml" field tags either.